frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -290,7 +290,7 @@ export const ImageManifestVulnList: React.FC<ImageManifestVulnListProps> = (prop
     <EmptyState variant={EmptyStateVariant.large}>
       <Title headingLevel="h4" size="lg">
         <EmptyStateResourceBadge model={ImageManifestVulnModel} />
-        {t('container-security~No Image Vulnerabilities found')}
+        {t('container-security~No image vulnerabilities found')}
```

Comment:
`image` can be title case `Image` here since Image is a proper noun in the OpenShift glossary.